components/inputs/input-date.js

```diff
@@ -20,9 +20,12 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 	static get properties() {
 		return {
 			disabled: { type: Boolean },
+			emptyStateText: { type: String, attribute: 'empty-state-text'},
```

Comment:
I'm OK with this name, or `empty-text` which I think is a term we've used before whenever we have no search results or no items in a list.